pkgs/development/python-modules/unicorn/default.nix

```diff
@@ -32,17 +35,17 @@ buildPythonPackage rec {
       "macosx_11_0"
     ];
 
-  build-system = [ setuptools ];
+  build-system = [
+    setuptools
+    setuptools-scm
+  ];
 
-  checkPhase = ''
-    runHook preCheck
+  nativeCheckInputs = [
+    capstone
+    pytestCheckHook
+  ];
 
-    mv unicorn unicorn.hidden
-    patchShebangs sample_*.py shellcode.py
-    sh -e sample_all.sh
-
-    runHook postCheck
-  '';
+  disabledTests = lib.optionals stdenv.hostPlatform.isx86_64 [ "test_i386" ];
```

Comment:
I think you meant to do the inverse of this?

And you can probably include 32bit by using `isx86` instead.

Reply:
Actually scratch that, I can't find a single platform `test_i386` works on.